controllers/storageclaim_controller.go

```diff
@@ -80,54 +111,74 @@ func (r *StorageClassClaimReconciler) SetupWithManager(mgr ctrl.Manager) error {
 			}
 			return []reconcile.Request{}
 		})
+	enqueueStorageClassClaimRequest := handler.EnqueueRequestsFromMapFunc(
+		func(_ context.Context, obj client.Object) []reconcile.Request {
+			return []reconcile.Request{{
+				NamespacedName: types.NamespacedName{
+					Name: obj.GetName(),
+				},
+			}}
+		},
+	)
```

Comment:
I am unable to follow your intention around this watch. why are we watching the old API and why are we queuing a request for a CR that might not even exist?

Reply:
But we are deprecating which means we should not act on any new storage class claim

Reply:
Will remove.

Reply:
done.